src/UI/TransactionsToastList/components/CustomToast/index.tsx

```diff
@@ -25,13 +32,22 @@ export const CustomToast = ({
     };
   }, []);
 
-  return (
-    <div className={wrapperStyles.toastWrapper}>
+  const closeButton = useMemo(() => {
```

Comment:
I don't think we need a useMemo here, it's not so performance intensive to do a ternary